src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  color: #000;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  background-color: #fff;
+}
+
+.shell {
+  max-width: 1200px;
+  width: 100%;
+  margin: 0 auto;
+  padding: 0 50px;
+}
+
+.header__shell {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```

Comment:
You can apply this styles to header, no need to create additional divs for such purpose